ravem/indico_ravem/controllers.py

```diff
@@ -25,58 +26,62 @@ def _check_access(self):
         if not has_access(self.event_vc_room):
             raise RavemException(_("Not authorized to access the room with RAVEM"))
 
+    def get_event_vc_room(self, _id):
+        event_vc_room = VCRoomEventAssociation.find_one(id=_id)
+        if not event_vc_room:
+            raise NotFound(_("Event VC Room not found for id {id}").format(id=_id))
```

Comment:
`find_one()` does not return if it didn't find exactly one row. But let's not use any of the `find` utils in new code, Use `VCRoomEventAssociation.query.get(_id)` instead (and rename `_id` to `id` (shadowing is OK here IMO) or `id_` - it's not internal)

Reply:
Alright, but why no `find` utils in the new code?

Reply:
indico/indico#4684